examples/passive2active.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+# Rewrite sentences in the passive voice to the active voice using the
+# 7B parameter Falcon model. This is only a small demo of using Curated
+# Transformers for a specific application of generation. The example only
+# works for relatively simple sentences, use larger models/better prompts/
+# finetuning to get better results.
+#
+# Example usage:
+#
+# ./passive2active.py < passive-examples.json
+#
+
+import argparse
+import json
+import sys
+from typing import List
+
+import torch
+
+from curated_transformers.generation.config import (
+    GreedyGeneratorConfig,
+    SampleGeneratorConfig,
+)
+from curated_transformers.generation.falcon import FalconGenerator
+from curated_transformers.layers.attention import enable_torch_sdp
+
+EPILOG = """This program takes passive sentences as a JSON list of strings
+from the standard inputs. For example:
+
+["The medal was won by the Dutch speed skater.", "Anita was driven to the theatre by Carla."]
+
+The output is a JSON list with the following format:
+
+[{"passive": "The medal was won by the Dutch speed skater.", "active": "..."},
+ {"passive": "Anita was driven to the theatre by Carla.", "active": "..."}]'"""
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+    description="Rewrite sentences in the active voice to passive voice",
+    epilog=EPILOG,
+)
+parser.add_argument(
```

Comment:
Shall we also add an argument to enable quantization?

Reply:
Great idea. Added. Also had to make `RefinedWebModel` inherit `Quantizable`.